Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalMinBias_Output_cff.py

```diff
@@ -10,10 +10,9 @@
     ),
     outputCommands = cms.untracked.vstring(
 	'keep *_gtDigisAlCaMB_*_*',
-        'keep HBHERecHitsSorted_hbherecoMB_*_*',
-        'keep HORecHitsSorted_horecoMB_*_*',
-        'keep HFRecHitsSorted_hfrecoMB_*_*',
-        'keep HFRecHitsSorted_hfrecoMBspecial_*_*',
```

Comment:
We do not produce the collection any more. The collections produced are with label NZS and these are the ones which are used in subsequent analysis. So this is a simple clean up process. The consequence of the removal is to remove misinformation which you fell for.